test/studies/lsms/shemmy/s-lsms.chpl

```diff
@@ -64,7 +64,7 @@ proc main() {
 
 	for itr in 0..#nIterations {
 		if itr % reportFrequency == 0 && reportFrequency <= nIterations {
-			writeln("step ", format("####", itr), "...");
+			writef("step  ###...\n", itr);
```

Comment:
It seems we lost a # here - I would have expected the writef to have #### like the original